garrysmod/gamemodes/terrortown/gamemode/cl_init.lua

```diff
@@ -368,6 +370,9 @@ function CheckIdle()
 
          timer.Simple(0.3, function()
                               RunConsoleCommand("ttt_spectator_mode", 1)
+                               net.Start("TTT_Spectate")
+                                 net.WriteBool(true)
+                               net.SendToServer()

```

Comment:
It means the player is respawned twice. Minor but it is something I was making sure was known.